### DISCUSSION TITLE
v1.98.0 - manage external libraries owners?

### DISCUSSION THREAD

**Discussion by aisbergde** [immich-app/immich#7743] [Q&A]
in v1.98.0 the external libraries have been moved into the administrator section. This is a right and very good decision.

libraries have owners, but it looks like the owner is read only
![image](https://github.com/immich-app/immich/assets/9020187/c8eb183f-f44f-4a57-8d41-c0d9e2995557)
And it looks like the owner is requested, when creating a new library.

Maybe the feature is WIP. And maybe this was just a migration from user settings into the administrator section.
And maybe there are already plans to implement user management for external libraries?

**Comment by alextran1502** [comment]
Yes, right now it is read-only, we are planning to use this as the foundation to add library's owner management in the future.

**Reply by LMSSonos** [reply]
@alextran1502 is there any way to move this manually to another user without lossing the face assignment? Asking because wanted to use this workaround https://github.com/immich-app/immich/discussions/4500#discussioncomment-10018274 to share external libraries
